### PR TITLE
Interactivity

### PR DESCRIPTION
**Pull request by Maedea9**
In this third part of the project I have:
- Created a new JavaScript file to be imported as a module with the "Clear all completed" button functionality.
- Added event listener to the checkbox with true/false updates.
- Stored the updated data in local storage.

### CODE REVIEW

File: src/modules/functionality.js [hunk 1]
```diff
@@ -1,70 +1,73 @@
...
-    taskBox.dataset.taskObject = JSON.stringify(taskObject);
     taskList.appendChild(taskBox);
+
+    // COMPLETED TASKS//
+    const checkbox = taskBox.querySelector('.input-check');
+    checkbox.addEventListener('click', (e) => {
+      const taskBox = e.target.parentElement.parentElement;
+      const taskIndex = Array.from(taskBox.parentElement.children).indexOf(taskBox);
+      const toDoArray = JSON.parse(localStorage.getItem('toDoArray')) || [];
+      toDoArray[taskIndex].completed = e.target.checked;
```
Comment (topeogunleye) [inline src/modules/functionality.js:26]: - [ ] Currently your updated array of items is stored in local storage but the correct list of values is not being reflected on the user interface after the page reloads. Kindly see my video below for more explanation.


https://www.loom.com/share/6eacc3e42fc84c6aa37f54bd34f926c1

File: src/modules/interactivity.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// CLEAR ALL COMPLETED//
+const clearAllCompleted = () => {
+  const toDoArray = JSON.parse(localStorage.getItem('toDoArray')) || [];
+  for (let i = 0; i < toDoArray.length; i += 1) {
+    if (toDoArray[i].completed) {
+      toDoArray.splice(i, 1);
```
[the remark this answers is not in the document]
Comment (Maedea9) [reply src/modules/interactivity.js:6]: ok! sure I'll implement it!

File: src/modules/functionality.js (hunk 1 above)
Comment (Maedea9) [reply src/modules/functionality.js:26]: @topeogunleye I missed that one! sorry! I'll fix it right away! thanks for taking the time to record a video to show me!

File: src/modules/interactivity.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Maedea9) [reply src/modules/interactivity.js]: Done!

**Pull request merged**